### PR TITLE
Added events_accel_tap_service_* functionality.

### PR DESCRIPTION
**Pull request by MorrisTimm**
Slow day at work, so I added the missing accel_tap_service stuff.
Only tested locally and not as part of the package but it worked for me.
Please review ;)

### CODE REVIEW

File: src/c/accel-tap.c [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#include <pebble.h>
+#include <@smallstoneapps/linked-list/linked-list.h>
+
+#include "pebble-events.h"
+
+static LinkedRoot *s_handler_list;
+static TimeUnits s_current_subscription;
```
Comment (Katharine) [inline src/c/accel-tap.c]: This probably shouldn't be here.

**Comment by MorrisTimm** [comment]
Agreed.

File: src/c/accel-tap.c (hunk 1 above)
Comment (MorrisTimm) [inline src/c/accel-tap.c]: Agreed. Copy & paste mistake.

**Comment by Katharine** [comment]
Adding the unobstructed area service means I can't land this because it would create a mess with pre-4.0 users, which are almost all of them.

**Comment by MorrisTimm** [comment]
Let's try this again with proper branching ...

**Pull request closed without merging**